### PR TITLE
Add support some DROP script

### PR DESCRIPTION
**Pull request by emin100**
Support deparse some DROP statement

DROP SERVER [ IF EXISTS ] server_name [ CASCADE | RESTRICT ]
DROP PUBLICATION [ IF EXISTS ] name [, ...] [ CASCADE | RESTRICT ]
DROP TYPE [ IF EXISTS ] name [, ...] [ CASCADE | RESTRICT ]
DROP TABLESPACE [ IF EXISTS ] name
DROP CONVERSION name [ CASCADE | RESTRICT ]
DROP SEQUENCE [ IF EXISTS ] name [, ...] [ CASCADE | RESTRICT ]

DROP INDEX [ CONCURRENTLY ] [ IF EXISTS ] name [, ...] [ CASCADE | RESTRICT ]
DROP MATERIALIZED VIEW [ IF EXISTS ] name [, ...] [ CASCADE | RESTRICT ]
DROP TRIGGER [ IF EXISTS ] name ON table_name [ CASCADE | RESTRICT ]
DROP RULE [ IF EXISTS ] name ON table_name [ CASCADE | RESTRICT ]
DROP CAST [ IF EXISTS ] (source_type AS target_type) [ CASCADE | RESTRICT ]
DROP OPERATOR FAMILY [ IF EXISTS ] name USING index_method [ CASCADE | RESTRICT ]
DROP AGGREGATE [ IF EXISTS ] name ( aggregate_signature ) [, ...] [ CASCADE | RESTRICT ]
DROP FUNCTION [ IF EXISTS ] name [ ( [ [ argmode ] [ argname ] argtype [, ...] ] ) ] [, ...]
    [ CASCADE | RESTRICT ]
DROP COLLATION [ IF EXISTS ] name [ CASCADE | RESTRICT ]
DROP FOREIGN DATA WRAPPER [ IF EXISTS ] name [, ...] [ CASCADE | RESTRICT ]
DROP ACCESS METHOD [ IF EXISTS ] name [ CASCADE | RESTRICT ]
DROP STATISTICS [ IF EXISTS ] name [, ...]
DROP TEXT SEARCH DICTIONARY [ IF EXISTS ] name [ CASCADE | RESTRICT ]
DROP OPERATOR CLASS [ IF EXISTS ] name USING index_method [ CASCADE | RESTRICT ]
DROP ROLE [ IF EXISTS ] name [, ...]

DROP POLICY [ IF EXISTS ] name ON table_name [ CASCADE | RESTRICT ]
DROP SUBSCRIPTION [ IF EXISTS ] name [ CASCADE | RESTRICT ]
DROP TEXT SEARCH CONFIGURATION [ IF EXISTS ] name [ CASCADE | RESTRICT ]
DROP FOREIGN TABLE [ IF EXISTS ] name [, ...] [ CASCADE | RESTRICT ]
DROP EVENT TRIGGER [ IF EXISTS ] name [ CASCADE | RESTRICT ]
DROP TEXT SEARCH PARSER [ IF EXISTS ] name [ CASCADE | RESTRICT ]
DROP VIEW [ IF EXISTS ] name [, ...] [ CASCADE | RESTRICT ]
DROP EXTENSION [ IF EXISTS ] extension_name [, ...] [ CASCADE | RESTRICT ]
DROP DOMAIN name [, ...]  [ CASCADE | RESTRICT ]
DROP TEXT SEARCH TEMPLATE [ IF EXISTS ] name [ CASCADE | RESTRICT ]
DROP TRANSFORM [ IF EXISTS ] FOR type_name LANGUAGE lang_name

### CODE REVIEW

**Comment by lfittl** [comment]
@emin100 Thanks again for all your contributions. As I'm going through them, I noticed a few PRs, including this one, didn't have tests.

Could you add tests to this PR for the statements that are now being supported? That'd be awesome :)

**Comment by emin100** [comment]
Tests is added

File: lib/pg_query/deparse.rb [hunk 1]
```diff
@@ -358,7 +364,8 @@ def deparse_alter_table_cmd(node)
 
     def deparse_object_with_args(node)
       output = []
-      output += node['objname'].map(&method(:deparse_item))
+      output << deparse_item_list(node['objname']).join('.')
+      # output += node['objname'].map(&method(:deparse_item_list)).join('.')
```
Comment (lfittl) [inline lib/pg_query/deparse.rb]: Can we remove this comment?

Comment (emin100) [reply lib/pg_query/deparse.rb]: Sure

Comment (emin100) [reply lib/pg_query/deparse.rb]: I remove the comment.

**Comment by lfittl** [comment]
@emin100 Thanks for the quick update!

Sorry again that it took a while to get to all of these, again appreciate your contributions :)

**Pull request merged**